src/App.vue

```diff
@@ -27,6 +28,9 @@ watch(isLoading, async (loading) => {
       <router-view v-if="isAuthenticated" />
     </main>
   </div>
+  <footer>
+    <media-player />
```

Comment:
should this have v-if="isAuthenticated"?

Reply:
I'm actually wondering whether we shouldn't replace the whole scaffolding with some loading indicator or so instead of checking it at each module ...